packages/manager/modules/nutanix/src/dashboard/general-info/redeploy/template.html

```diff
@@ -239,7 +240,9 @@ <h2 data-translate="nutanix_redeploy_title"></h2>
                         data-ng-pattern="$ctrl.IPV4_REGEX"
                     />
                 </oui-field>
-                <div>
+                <div
+                    data-ng-if="$ctrl.config.prismCentral.type !== $ctrl.PRISM_CENTRAL_TYPE_ALONE"
```

Comment:
Could we use some sort of model instead of making this kind of checks in the views ?

Reply:
Can you have a look on the PR https://github.com/ovh/manager/pull/6706 ?

cc- @marie-j